manifests/config.pp

```diff
@@ -1,7 +1,9 @@
 # Configure dns
 # @api private
 class dns::config {
-  group { $dns::params::group: }
+  if $dns::group_manage == true {
```

Comment:
```suggestion
  if $dns::group_manage {
```

Reply:
@ekohl Thanks for the comment! I have updated the PR and removed the explicit comparison.